.migration-backup/ui-components/src/ui/atoms/Avatar/Avatar.stories.tsx

```
@@ -0,0 +1,268 @@
+import { logger } from '../../../utils/logger';
```

ESLint

[error] '../../../utils/logger' import is restricted from being used by a pattern. Avoid deep relative imports. Use path aliases (@ui, @domains, @lib, etc.) instead.

(no-restricted-imports)